events/wc_order_events.txt

```diff
@@ -906,31 +906,136 @@ narrative_event = {
 		name = EVTOPTB_WCORD_406
 	}
 }
+# The Creation of Blood Knights Order
+namespace = WCORD
+narrative_event = {
+	id = WCORD.74
```

Comment:
The event ID above is WCORD.406, the event ID below is WCORD.9000. A question, why it''s 74?

Reply:
Just...Because?

Reply:
Can you change it to... WCORD.500 for example? :3